src/foam/nanos/auth/ClientAgentAuthService.js

```diff
@@ -0,0 +1,37 @@
+foam.CLASS({
+  package: 'foam.nanos.auth',
+  name: 'ClientAgentAuthService',
+
```

Comment:
Please document this class.

Reply:
The client agent auth service? Not sure if it makes much sense to document. I've documented the agent auth service interface which should be fine considering it contains the service methods and is displayed in the generated api documentation.